### PR TITLE
Node 14 updates.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: openshift/4.0/templates/jenkins-slaves/README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
 PIMS Jenkins pipelines requires 3 Jenkins agents to execute.
 
-1. [NPM](./jenkins-slave-npm/README.md) - for building the frontend application
```
Comment (devinleighsmith) [inline openshift/4.0/templates/jenkins-slaves/README.md:3]: As far as I'm aware we have removed the usage of these agents (favoring github actions) and have no intention of re-introducing them

File: frontend/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM node:10-slim as node
+FROM node:14-slim as node
```
Comment (devinleighsmith) [inline frontend/Dockerfile:1]: I needed to recreate my container for this to work properly

**Review by asanchezr — APPROVED** [review]

**Review by jdtoombs — APPROVED** [review]

**Comment by devinleighsmith** [comment]
@Fosol  I may need you to override this one, the GH action for the node 10.x build is never run here (by design 14.x is the replacement). However, GH is listing its completion as a requirement still.

**Comment by Fosol** [comment]
@devinleighsmith I don't have that ability here.

**Comment by devinleighsmith** [comment]
@sookeke or @DarylTodosichuk or you able to force merge this? As I mentioned above, the build10.x will never pass for this PR, as it has been replaced by the build14.x . Once this PR is merged, all future PRs will use the build 14.x instead.

**Comment by sookeke** [comment]
@Fosol can you confirm this?

**Comment by Fosol** [comment]
Please merge.

**Pull request merged**